analyzer/codechecker_analyzer/checkers.py

```diff
@@ -23,7 +23,8 @@ def available(ordered_checkers, available_checkers):
         if checker_name.startswith('profile:') or \
                 checker_name.startswith('guideline:') or \
                 checker_name.startswith('severity:') or \
-                checker_name.startswith('sei-cert:'):
+                checker_name.startswith('sei-cert:') or \
+                checker_name.startswith('group:'):
```

Comment:
If you plan to introduce a new checker enable prefix, please add this to the documentation and help of CodeChecker analyze and CodeChecker check.

Reply:
Will do. Thanks!